src/domains/chart/sagas.ts

```diff
@@ -135,7 +135,7 @@ function* fetchDataSaga({ payload }: Action<FetchDataPayload>) {
     dimensions,
     context: chart,
     ...(aggrMethod && { aggr_method: aggrMethod }),
-    ...(nodeIDs && { node_ids: nodeIDs.join(",") }),
+    ...(nodeIDs && nodeIDs.length && { node_ids: nodeIDs.join(",") }),
```

Comment:
Hmmm @jacekkolasa @Azein @novykh  
This means that if the filter returns no Node it will show all Nodes? Is this the expected behaviour?

Reply:
i was worried about it, too, but @lvrach confirmed that no node-ids (`&node_ids=` at the end) will have the same result as not providing node_ids at all. Makes sense for me, but i agree we need to be careful here